lib/python/rosie/vc.py

```diff
@@ -464,6 +531,40 @@ def create(argv):
             info_config = rose.config.load(info_file.name)
         finally:
             os.unlink(info_file.name)
+        new_id = "Not yet"
```

Comment:
An empty string may be clearer than a magic string? Otherwise, the magic string should probably be a constant.